deploy/crds/nmstate.io_nmstates_crd.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: nmstates.nmstate.io
+spec:
+  group: nmstate.io
+  names:
+    kind: Nmstate
+    listKind: NmstateList
+    plural: nmstates
+    singular: nmstate
+  scope: Namespaced
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      description: Nmstate is the Schema for the nmstates API
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          description: NmstateSpec defines the desired state of Nmstate
+          properties:
+            replicas:
+              format: int32
+              type: integer
+          type: object
+        status:
+          description: NmstateStatus defines the observed state of Nmstate
+          properties:
+            nodes:
+              description: 'INSERT ADDITIONAL STATUS FIELD - define observed state
```

Comment:
Please don't keep placeholders in the code. Let's either define some useful properties or leave it blank.